newgame/src/resources/cargo_deps.txt

```diff
@@ -0,0 +1,10 @@
+serde = "*"
+serde_json = "*"
+serde_derive = "*"
+lazy_static = "*"
+rand = { version = "0.6.3", default-features = false, features=["std"] }
```

Comment:
Can we turn these version numbers into "*" and have it be OK? idk, but it would be nice to be less-specific here...

Reply:
I copied the other `Cargo.toml`s here; presumably you chose a specific version for a reason?

Reply:
Yeah, I got paranoid with rand because they don't consider the specific random number generator to be part of semver... and we really want to be as reproducible as possible with a given seed.